### PR TITLE
Update id.m3u

### PR DESCRIPTION
**Pull request by leap0x7b**

### CODE REVIEW

File: streams/id.m3u [hunk 1]
```diff
@@ -1,12 +1,14 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="ANTV.id" status="online",ANTV (720p)
+http://210.210.155.35/qwr9ew/s/s07/index1.m3u8
 #EXTINF:-1 tvg-id="BeritaSatu.id" status="online",BeritaSatu (540p)
 http://edge.linknetott.swiftserve.com/live/BsNew/amlst:beritasatunewsbs/playlist.m3u8
 #EXTINF:-1 tvg-id="BeritaSatu.id" status="online",BeritaSatu (720p)
 https://b1news.beritasatumedia.com/Beritasatu/B1News_manifest.m3u8
-#EXTINF:-1 tvg-id="BeritaSatuEnglish.id" status="online",BeritaSatu English (540p) [Not 24/7]
-http://edge.linknetott.swiftserve.com/live/BsNew/amlst:bsenglish/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:7]: Works for me.

File: streams/id.m3u [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 #EXTINF:-1 tvg-id="BeritaSatuWorld.id" status="online",BeritaSatu World (540p) [Not 24/7]
 http://edge.linknetott.swiftserve.com/live/BsNew/amlst:bsnew/playlist.m3u8
+#EXTINF:-1 tvg-id="BeritaSatuWorld.id" status="online",BeritaSatu World (720p) [Not 24/7]
+https://b1world.beritasatumedia.com/Beritasatu/B1World_1280x720.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:11]:
```suggestion
https://b1world.beritasatumedia.com/Beritasatu/B1World_manifest.m3u8
```

File: streams/id.m3u [hunk 3]
```diff
@@ -23,22 +25,26 @@ https://liveanevia.mncnow.id/live/eds/HitsMovies/sa_dash_vmx/HitsMovies.mpd
 http://iamchannel.org:1935/tes/1/playlist.m3u8
 #EXTINF:-1 tvg-id="IndonesiaChannel.id" status="online",Indonesia Channel (720p) [Not 24/7]
 http://wowzaprod236-i.akamaihd.net/hls/live/1019903/7dd4cf51/playlist.m3u8
-#EXTINF:-1 tvg-id="Indosiar.id" status="blocked",Indosiar [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/205-indosiar
-#EXTINF:-1 tvg-id="" status="online",iNews (720p)
+#EXTINF:-1 tvg-id="Indosiar.id" status="online",Indosiar (720p)
```
Comment (LaneSh4d0w) [inline streams/id.m3u:28]: Doesn't work for me. Might be geo-blocked.

File: streams/id.m3u [hunk 4]
```diff
@@ -23,22 +25,26 @@ https://liveanevia.mncnow.id/live/eds/HitsMovies/sa_dash_vmx/HitsMovies.mpd
...
+http://rr2.dens.tv/s/s04/01.m3u8
+#EXTINF:-1 tvg-id="INews.id" status="online",iNews (720p)
 https://vcdn2.rctiplus.id/live/eds/inews_fta/live_fta/inews_fta.m3u8
-#EXTINF:-1 tvg-id="KompasTV.id" status="error",Kompas TV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/874-kompas-tv
+#EXTINF:-1 tvg-id="BeritaSatuEnglish.id" status="online",JakartaGlobe News (540p) [Not 24/7]
```
Comment (LaneSh4d0w) [inline streams/id.m3u:32]: Why is the name not corresponding to the tvg-id ?

File: streams/id.m3u [hunk 5]
```diff
@@ -23,22 +25,26 @@ https://liveanevia.mncnow.id/live/eds/HitsMovies/sa_dash_vmx/HitsMovies.mpd
...
-#EXTINF:-1 tvg-id="NetTV.id" status="error",Net. TV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/875-net-tv
-#EXTINF:-1 tvg-id="OChannel.id" status="error",O Channel [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/206-ochannel
+#EXTINF:-1 tvg-id="MyTV.id" status="online",MyTV (720p) [Not 24/7]
+http://playlist.urfan.web.id/x6bnqe/s/s88/S4/mnf.m3u8
+#EXTINF:-1 tvg-id="NetTV.id" status="online",Net. TV (720p) [Not 24/7]
+http://45.126.83.51/qwr9ew/s/s08/01.m3u8
+#EXTINF:-1 tvg-id="OChannel.id" status="online",O Channel (720p)
+http://rr3.dens.tv/s/s73/01.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:47]: Doesn't work for me.

File: streams/id.m3u [hunk 6]
```diff
@@ -23,22 +25,26 @@ https://liveanevia.mncnow.id/live/eds/HitsMovies/sa_dash_vmx/HitsMovies.mpd
...
 #EXTINF:-1 tvg-id="MetroTV.id" status="online",Metro TV (360p) [Not 24/7]
 http://edge.metrotvnews.com:1935/live-edge/smil:metro.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="MNCTV.id" status="online",MNCTV (720p)
 https://vcdn2.rctiplus.id/live/eds/mnctv_fta/live_fta/mnctv_fta.m3u8
-#EXTINF:-1 tvg-id="MyTV.id" status="error",MyTV (720p) [Not 24/7]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/6711-mytv
-#EXTINF:-1 tvg-id="NetTV.id" status="error",Net. TV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/875-net-tv
-#EXTINF:-1 tvg-id="OChannel.id" status="error",O Channel [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/206-ochannel
```
Comment (LaneSh4d0w) [inline streams/id.m3u:41]: Over local verification, seems to work. I don't see why it's taken off.

File: streams/id.m3u [hunk 7]
```diff
@@ -49,16 +55,20 @@ https://vcdn2.rctiplus.id/live/eds/rcti_fta/live_fta/rcti_fta.m3u8
 http://edge.linknetott.swiftserve.com/live/BsNew/amlst:reformedch/playlist.m3u8
 #EXTINF:-1 tvg-id="RRINet.id" status="online",RRI Net (720p) [Not 24/7]
 http://36.89.47.217:11935/rrinet/live/index.m3u8
-#EXTINF:-1 tvg-id="RTV.id" status="error",RTV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/1561-rtv
-#EXTINF:-1 tvg-id="SCTV.id" status="error",SCTV (480p) [Not 24/7]
-https://liveanevia.mncnow.id/live/eds/SCTV/sa_dash_vmx/SCTV.mpd
+#EXTINF:-1 tvg-id="RTV.id" status="online",RTV (720p)
+https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(41869351-4a9c-4e57-b54b-8cf5dfc919e1)/index.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:59]: Geo-blocked.

File: streams/id.m3u [hunk 8]
```diff
@@ -49,16 +55,20 @@ https://vcdn2.rctiplus.id/live/eds/rcti_fta/live_fta/rcti_fta.m3u8
...
+#EXTINF:-1 tvg-id="SCTV.id" status="online",SCTV (720p) [Not 24/7]
+http://rr2.dens.tv/s/s03/01.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:61]: Doesn't work for me.

File: streams/id.m3u [hunk 9]
```diff
@@ -49,16 +55,20 @@ https://vcdn2.rctiplus.id/live/eds/rcti_fta/live_fta/rcti_fta.m3u8
...
+#EXTINF:-1 tvg-id="SeaToday.id" status="online",Sea Today (720p)
+https://cdn-telkomsel-01.akamaized.net/Content/DASH/Live/channel(9ce3f094-4044-467e-84b7-b684a49571d5)/manifest.mpd
```
Comment (LaneSh4d0w) [inline streams/id.m3u:63]: Geo-blocked

File: streams/id.m3u [hunk 10]
```diff
@@ -49,16 +55,20 @@ https://vcdn2.rctiplus.id/live/eds/rcti_fta/live_fta/rcti_fta.m3u8
...
 #EXTINF:-1 tvg-id="Trans7.id" status="online",Trans7 (720p) [Not 24/7]
 https://video.detik.com/trans7/smil:trans7.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="TransTV.id" status="online",Trans TV (720p)
 https://video.detik.com/transtv/smil:transtv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="TVKU.id" status="online",TVKU (720p) [Not 24/7]
 http://103.30.1.14:8080/hls/live.m3u8
+#EXTINF:-1 tvg-id="TVOne.id" status="online",TVOne (720p)
+http://rr2.dens.tv/s/s105/01.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:71]: Doesn't work for me.

File: streams/id.m3u (hunk 4 above)
Comment (leapofazzam123) [reply streams/id.m3u:32]: BetitaSatu English was renamed to JakartaGlobe News. Its intended for compatibility

File: streams/id.m3u (hunk 6 above)
Comment (leapofazzam123) [reply streams/id.m3u:41]: Screenshot_20220308-203310

Comment (LaneSh4d0w) [reply streams/id.m3u:41]: Well I tried using my Windows streamlink and it works. Maybe it's due to something else.

Only @dct-infra can tell us what was going on. I know lots of Indonesians / Malaysians used my old "query-streamlink" heroku service for getting access to these streams, and it sometimes had cut off. Maybe it's due to rate limiting or throttling too.

Comment (dct-infra) [reply streams/id.m3u:41]: @LaneSh4d0w The front is currently russian but it changes randomly every other week or month.
The QS however is multi-homed and queries can be made from 2 to several EU or russian locations, whether round robin or specifically chosen, it depends on rate limiting and other usual issues.
QS logs show code 200 on several EU backends.

**Review by its0x08 — DISMISSED** [review]

File: streams/id.m3u (hunk 6 above)
Comment (leapofazzam123) [reply streams/id.m3u:41]: I guess the Russian server down due to the Russia-Ukraine situation

Comment (LaneSh4d0w) [reply streams/id.m3u:41]: > @LaneSh4d0w The front is currently russian but it changes randomly every other week or month. The front is invisible to destination servers (YT, DLM, Twitch etc) most of the time, except when a QS is active on the current front. So QS is multi-homed and queries can be made from 2 to several EU or russian locations, whether round robin or specifically chosen, it depends on rate limiting and other usual issues. see the picture? Concerning our issue, QS logs show code 200 on several backends for the listed channels, which is not a proof of a successful query in QS world. A simple wide [Geo-Block] imho.

Most certainly this happening. I was wondering about a proxy pool for certain DM geo-blocked links.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check all the links in the request again.

**Comment by leapofazzam123** [comment]
https://github.com/iptv-org/iptv/pull/6628#discussion_r830026085
@freearhey should work now

**Comment by freearhey** [comment]
@leapofazzam123 I was talking about this link

```
http://rr3.dens.tv/s/s73/01.m3u8
```

**Comment by leapofazzam123** [comment]
should work now

**Review by freearhey — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
From ANTV host : 

- TransTV / http://210.210.155.35/qwr9ew/s/s05/index.m3u8
- SCTV / http://210.210.155.35/qwr9ew/s/s03/index.m3u8

File: streams/id.m3u [hunk 11]
```diff
@@ -1,18 +1,20 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="ANTV.id" status="blocked",ANTV (720p) [Geo-blocked]
+http://210.210.155.35/qwr9ew/s/s07/index1.m3u8
 #EXTINF:-1 tvg-id="BeritaSatu.id" status="online",BeritaSatu (540p)
 http://edge.linknetott.swiftserve.com/live/BsNew/amlst:beritasatunewsbs/playlist.m3u8
 #EXTINF:-1 tvg-id="BeritaSatu.id" status="online",BeritaSatu (720p)
 https://b1news.beritasatumedia.com/Beritasatu/B1News_manifest.m3u8
-#EXTINF:-1 tvg-id="BeritaSatuEnglish.id" status="online",BeritaSatu English (540p) [Not 24/7]
-http://edge.linknetott.swiftserve.com/live/BsNew/amlst:bsenglish/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:7]: If this channel still works for us, why is it still deleted ?

File: streams/id.m3u [hunk 12]
```diff
@@ -1,18 +1,20 @@
...
 http://edge.linknetott.swiftserve.com/live/BsNew/amlst:bsnew/playlist.m3u8
+#EXTINF:-1 tvg-id="BeritaSatuWorld.id" status="online",BeritaSatu World (720p) [Not 24/7]
+https://b1world.beritasatumedia.com/Beritasatu/B1World_manifest.m3u8
 #EXTINF:-1 tvg-id="CNBCIndonesia.id" status="online",CNBC Indonesia (720p)
 https://live.cnbcindonesia.com/livecnbc/smil:cnbctv.smil/master.m3u8
 #EXTINF:-1 tvg-id="CNNIndonesia.id" status="online",CNN Indonesia (720p)
 https://live.cnnindonesia.com/livecnn/smil:cnntv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="GTV.id" status="online",GTV (720p)
-https://vcdn2.rctiplus.id/live/eds/gtv_fta/live_fta/gtv_fta.m3u8
+https://goldenlivektv.com/Live_TV/Channel_Gtv.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:17]: Redirects to https://gtv-linier.rctiplus.id/live/eds/GTV2021/live_fta/index.m3u8

File: streams/id.m3u [hunk 13]
```diff
@@ -23,42 +25,50 @@ https://liveanevia.mncnow.id/live/eds/HitsMovies/sa_dash_vmx/HitsMovies.mpd
...
-#EXTINF:-1 tvg-id="Indosiar.id" status="blocked",Indosiar [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/205-indosiar
-#EXTINF:-1 tvg-id="" status="online",iNews (720p)
-https://vcdn2.rctiplus.id/live/eds/inews_fta/live_fta/inews_fta.m3u8
-#EXTINF:-1 tvg-id="KompasTV.id" status="error",Kompas TV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/874-kompas-tv
+#EXTINF:-1 tvg-id="Indosiar.id" status="blocked",Indosiar (720p) [Geo-blocked]
+http://rr2.dens.tv/s/s04/01.m3u8
+#EXTINF:-1 tvg-id="INews.id" status="online",iNews (720p)
+https://goldenlivektv.com/Live_TV/Channel_Inews.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:31]: Redirects to https://inews-linier.rctiplus.id/live/eds/INEWS_2021/live_fta/index.m3u8

File: streams/id.m3u [hunk 14]
```diff
@@ -23,42 +25,50 @@ https://liveanevia.mncnow.id/live/eds/HitsMovies/sa_dash_vmx/HitsMovies.mpd
...
 http://edge.metrotvnews.com:1935/live-edge/smil:metro.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="MNCTV.id" status="online",MNCTV (720p)
-https://vcdn2.rctiplus.id/live/eds/mnctv_fta/live_fta/mnctv_fta.m3u8
-#EXTINF:-1 tvg-id="MyTV.id" status="error",MyTV (720p) [Not 24/7]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/6711-mytv
-#EXTINF:-1 tvg-id="NetTV.id" status="error",Net. TV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/875-net-tv
-#EXTINF:-1 tvg-id="OChannel.id" status="error",O Channel [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/206-ochannel
+https://goldenlivektv.com/Live_TV/Channel_Mnctv.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:41]: Redirects to https://mnctv-linier.rctiplus.id/live/eds/MNCTV_2021/live_fta/index.m3u8

**Comment by leapofazzam123** [comment]
> From ANTV host :
> 
>     * TransTV / http://210.210.155.35/qwr9ew/s/s05/index.m3u8
> 
>     * SCTV / http://210.210.155.35/qwr9ew/s/s03/index.m3u8
> 
>     * NET. / http://210.210.155.35/qwr9ew/s/s08/index.m3u8
> 
>     * Elshinta TV / http://210.210.155.35/qwr9ew/s/s10/index.m3u8
> 
>     * TVRI / http://210.210.155.35/qwr9ew/s/s11/index.m3u8
> 
>     * DAAI TV / http://210.210.155.35/qwr9ew/s/s13/index.m3u8
> 
> 
> I should analyse this host as it's an Indonesian stream pile, some aren't even referenced in this playlist ! :)

It's actually from the same Dens.TV (r2.dens.tv) host

**Review by freearhey — DISMISSED** [review]

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Fixed the issue about this PR, should be OK to push.

**Review by freearhey — APPROVED** [review]

**Pull request merged**